### DISCUSSION TITLE
Senha do funcionário

### DISCUSSION THREAD

**Discussion by lucassl2020** [EuricoDNJR/SG-VILLA-DOLCE#78] [Q&A] [answered]
O gerente pode ver a senha do funcionário ao acessar as informações do funcionário?

Se não pode, como um funcionário pode editar sua senha? Caso não seja possível ver e editar a senha na pagina de funcionários?

**Comment by EuricoDNJR** [comment, marked as answer]
- O gerente não pode ver a senha do usuário.
- O funcionário não pode alterar suas informações, caso seje necessário, ele deve solicitar ao gerente.
